.github/workflows/latex-linter.yaml

```diff
@@ -0,0 +1,13 @@
+name: run-linters
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+    paths: ["docs/**/**.tex"]
+jobs:
+  run-linter:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions-rust-lang/setup-rust-toolchain@v1
+      - run: cargo install tex-fmt
+      - run: tex-fmt --check ./docs/**/**.tex
```

Comment:
aight I'm prolly dumb, but when I did ./docs/**.tex it didn't check subdirectories, so maybe I'm a noob, but Idk I think I'm just gonna stick with this and we can fix it later